tools/vagrant.md

```diff
@@ -24,7 +24,7 @@ COMMAND | DESCRIPTION
 ---|---
 `vagrant provision` |  Forces reprovisioning of the vagrant machine
 `vagrant provision --debug ` | Use the debug flag to increase the verbosity of the output
-`vagrant up --provision | tee provision.log` | Runs `vagrant up`, forces provisioning and logs all output to a file
+`vagrant up --provision \| tee provision.log` | Runs `vagrant up`, forces provisioning and logs all output to a file
```

Comment:
@zairahira I did not add a backtick. I added a backslash **only** to escape the pipe because it breaks the rendered table, see https://github.com/ChristianLempa/cheat-sheets/blob/6020a1e39a4d0ed294a8cbff7a4b1e1b352eb287/tools/vagrant.md#provisioning-vms

Inside tables, pipe characters are required to be escaped, see also [example 200](https://github.github.com/gfm/#example-200) of the official GitHub Flavored Markdown Spec.

Reply:
You're right, I misinterpreted the syntax.